apps/chat/src/store/import-export/importExport.epics.ts

```diff
@@ -167,14 +254,46 @@ const importConversationsEpic: AppEpic = (action$, state$) =>
         return of(ImportExportActions.resetState());
       }
 
-      return concat(
-        of(ImportExportActions.importConversationsSuccess()),
-        of(
-          ConversationsActions.importConversationsSuccess({
-            conversations: history,
-            folders,
-          }),
+      const conversationsInfoToUpload = history.filter((conversation) => {
+        return conversation.status !== 'LOADED';
```

Comment:
```suggestion
        return conversation.status !== 'LOADED';
```

```suggestion
        return conversation.status !== UploadStatus.LOADED;
```
Don't forget to import this

Reply:
Changed to 'UploadStatus.LOADED'!